{{cookiecutter.project_slug}}/frontend/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
copy pasted from react app

Reply:
You can remove this, we don't use it with Tilt and k8s.